backend/lib/spree/backend_configuration.rb

```diff
@@ -6,6 +6,22 @@ module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: I18n.default_locale
 
+    # @!attribute [rw] themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :themes, :hash, default: {
+      classic: 'spree/backend/all',
+    }
+
+    FALLBACK_THEME = :classic
+
+    # @!attribute [rw] theme
+    #   @return [String] Default admin light theme name
+    preference :theme, :string, default: :classic
+
+    def theme_path
+      themes.fetch(theme.to_sym) { themes.fetch(:classic) }
```

Comment:
Given that `theme` will return `:classic` by default, probably there's no need to provide a block to `fetch`.